### PR TITLE
Project action

### LINKED ISSUE #2503: Project action
**Issue summary**
Automate adding issues to the project using https://github.com/actions/add-to-project

**Other information and links**

### PR DESCRIPTION
**Pull request by tyshko5**
## Summary of changes

Changes introduced in this pull request:
- 

## Reference issue to close (if applicable)

Closes #2503

## Change checklist

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works (if possible),
- [x] I have made sure the [CHANGELOG](https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md) is up-to-date. All user-facing changes should be reflected in this document.

### CODE REVIEW

**Comment by tyshko5** [comment]
@lerajk Here's the PR for the action

**Comment by LesnyRumcajs** [comment]
@tyshko5 could you please show an example of how it works and how it populates the board? E.g, by creating a separate repository with this workflow (or forking Forest I guess)

**Comment by tyshko5** [comment]
> @tyshko5 could you please show an example of how it works and how it populates the board? E.g, by creating a separate repository with this workflow (or forking Forest I guess)

I tested it here https://github.com/tyshko5/test_actions @LesnyRumcajs

**Comment by LesnyRumcajs** [comment]
> > @tyshko5 could you please show an example of how it works and how it populates the board? E.g, by creating a separate repository with this workflow (or forking Forest I guess)
> 
> I tested it here https://github.com/tyshko5/test_actions @LesnyRumcajs

Can you provide the link to the filled board?

**Comment by tyshko5** [comment]
> > > @tyshko5 could you please show an example of how it works and how it populates the board? E.g, by creating a separate repository with this workflow (or forking Forest I guess)
> > 
> > 
> > I tested it here https://github.com/tyshko5/test_actions @LesnyRumcajs
> 
> Can you provide the link to the filled board?

https://github.com/users/tyshko5/projects/1

**Comment by LesnyRumcajs** [comment]
@tyshko5 the board may be private, I get 404.

image

**Comment by tyshko5** [comment]
> @tyshko5 the board may be private, I get 404.
> 
> image

made it public

**Comment by tyshko5** [comment]
> @tyshko5 what about the dates we talked about? The issue should have a date on the board when it was closed.

@LesnyRumcajs Looks like there is no way to fill the columns of any kind. The action can just add to the board based of an event and labels and that's it. https://github.com/tyshko5/test_actions/actions/runs/4127203186/jobs/7130118472#step:2:1

**Comment by LesnyRumcajs** [comment]
If @lerajk is okay with this and filling the dates manually, then I'm fine. Otherwise, there's a way, but it's *a bit* more complex to implement. See the example [here](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/automating-projects-using-actions). Most likely we could snatch it and customize it to our needs.

File: .github/workflows/project.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Add bugs to bugs project
```
[the remark this answers is not in the document]
Comment (tyshko5) [reply .github/workflows/project.yml]: Fixed

**Review by lemmih — APPROVED** [review]

**Pull request merged**